### PR TITLE
Entry point made mandatory

### PR DESCRIPTION
**Pull request by sshveta**
{{pytest: -v --long-running cfme/tests/services/test_generic_service_catalogs.py }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 56a3b1de448112f3d4f4f312598a31b8bf02323e:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 53bc0d576c55f854f84928a32ef43eadf8768f34:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/services/catalogs/catalog_item.py [hunk 1]
```diff
@@ -209,7 +208,7 @@ def create(self):
                                'select_dialog': self.dialog,
                                'select_orch_template': self.orch_template,
                                'select_provider': self.provider_type})
-        if (current_version() >= "5.4") and (self.item_type != "Orchestration"):
+        if (self.item_type != "Orchestration"):
```
Comment (mfalesni) [inline cfme/services/catalogs/catalog_item.py]: Please, remove the parentheses.

**Comment by dajoRH** [comment]
Lint report for commit fb0febf7fa7922b7223325b363c5d49aed3fbaf0:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a0a5d6e05c5d6e050e4f311f0daea087bb89a69d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8b28f5d70296aca464323e27206c018fd2ac59f2:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 7a979d08fe6d9b7428314eadbd75705e8bf92d6b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**